samples/swift/FirebaseUI-demo-swift/Samples/StorageViewController.swift

```diff
@@ -50,12 +54,19 @@ class StorageViewController: UIViewController {
     self.storageRef = Storage.storage().reference(withPath: url.path)
 
     self.imageView.sd_setImage(with: self.storageRef,
-      placeholderImage: nil) { (image, error, cacheType, storageRef) in
+                               maxImageSize: 10000000,
+                               placeholderImage: nil,
+                               options: [.progressiveLoad]) { (image, error, cacheType, storageRef) in
       if let error = error {
         print("Error loading image: \(error)")
       }
     }
   }
+  
+  @objc fileprivate func flushCache() {
```

Comment:
nit: this method can probably be private instead of fileprivate.

Reply:
Seems other target action method use fileprivate (like the above `loadButtonPressed`) as well 😅 Change them at the same time ?

Reply:
You can leave this as-is then, I will address them separately later.

Reply:
Done.

Actually in daily Swift demo code, I usually leave it as the default `func`, which is internel level control. Isn't that suitable ?